source/agora/consensus/SlashPolicy.d

```diff
@@ -40,7 +40,8 @@ import agora.consensus.state.UTXODB;
 import agora.utils.Log;
```

Comment:
The comment in the commit should be changed, because the preimage root is not merkleized. It's just a hash concatenation of all the preimages which were sorted by the utxo key.